openwisp_firmware_upgrader/tests/test_api.py

```diff
@@ -29,6 +29,7 @@ class TestAPIUpgraderMixin(TestMultitenantAdminMixin, TestUpgraderMixin):
     def setUp(self):
         self.org = self._get_org()
         self.operator = self._create_operator(organizations=[self.org])
+        self.operator.organizations_dict  # force caching
```

Comment:
What do you think about this hack? I did it because using organizations_dict in OrgAPIMixin, makes an extra query than previously. I haven't come down to the reason.

Do you know if there was some cache involved?

```diff
-            user_orgs = self.request.user.openwisp_users_organization.all()
+            user_orgs = self.request.user.organizations_dict.keys()
```